### PR TITLE
build compat templates for Django 1.4-1.10

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/2049

### CODE REVIEW

**Comment by seandst** [comment]
Out of curiosity, is the version-blocking necessary to prevent gnarly DeprecationWarnings (or something?). If not, is it reasonable to have `TEMPLATES` alongside `TEMPLATE_DIRS` and `TEMPLATE_DEBUG` in the same evaluated settings file unconditionally, and have the values defined in one place and referenced from the other?

**Comment by asmacdo** [comment]
ok test

**Comment by asmacdo** [comment]
ok test

**Comment by asmacdo** [comment]
@seandst, yes that works!

File: server/pulp/server/content/web/settings.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import logging
 import os
 
+import django
```
Comment (dkliban) [inline server/pulp/server/content/web/settings.py]: why is this import needed?

**Comment by asmacdo** [comment]
ok test
